decentraland/unity-explorer
# fix: update avatar animations to fix neck behaviour

## What does this PR change?


Fix #2072 

Updated all avatar animations with the provided file.

The file provided in the issue wasn't working. 
[This](https://github.com/user-attachments/files/17127489/Avatar_Locomotion_Fix.glb.zip) is the file that was used.


## How to test the changes?



1. Launch the explorer
2. Check that the avatar has no longer any issue with the neck (was very visibile while wearing the white/blu shirt with the necklace)
3. Check all the animation interactions (idle, walk, jump, run, etc...)

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
